experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -320,7 +332,8 @@ function Interfacer.update_field!(sim::ClimaAtmosSimulation, ::Val{:turbulent_fl
     surface_normal = @. CA.C3(CA.unit_basis_vector_data(CA.C3, surface_local_geometry))
 
     # get template objects for the contravariant components of the momentum fluxes (required by Atmos boundary conditions)
-    vec_ct12_ct1 = @. CA.CT12(CA.CT2(CA.unit_basis_vector_data(CA.CT1, surface_local_geometry)), surface_local_geometry)
+    # NOTE: This is allocating! Fix me if you can!
```

Comment:
We can probably use the tensor_from_components function in ClimaAtmos, not sure it can fix the allocation. (You don't need to change it in this PR, just a comment.)